pkg/gossiper/db.go

```diff
@@ -67,3 +68,33 @@ func (d *mydb) ListData() (int, []pair) {
 	}
 	return SUCCEED, ans
 }
+
+type mybatch struct {
```

Comment:
Could just name this batch
```suggestion
type batch struct {
```